app/src/main/java/org/wikipedia/page/PageFragmentLoadState.java

```diff
@@ -181,27 +184,38 @@ private void pageLoadFromNetwork(final ErrorCallback errorCallback) {
 
         app.getSessionFunnel().leadSectionFetchStart();
 
-        disposables.add(ServiceFactory.getRest(model.getTitle().getWikiSite())
+        disposables.add(Observable.zip(ServiceFactory.getRest(model.getTitle().getWikiSite())
                 .getSummaryResponse(model.getTitle().getPrefixedText(), null, model.getCacheControl().toString(),
                         model.shouldSaveOffline() ? OfflineCacheInterceptor.SAVE_HEADER_SAVE : null,
-                        model.getTitle().getWikiSite().languageCode(), UriUtil.encodeURL(model.getTitle().getPrefixedText()))
+                        model.getTitle().getWikiSite().languageCode(), UriUtil.encodeURL(model.getTitle().getPrefixedText())), ServiceFactory.get(model.getTitle().getWikiSite())
```

Comment:
Can this somehow be done only if the user is logged in?

Reply:
Done!